packages/xo-server/src/vhd-merge.js

```diff
@@ -742,18 +822,40 @@ export async function chainVhd (
     parentVhd.readHeaderAndFooter(),
     childVhd.readHeaderAndFooter(),
   ])
-
-  const { header } = childVhd
+  await childVhd.readBlockTable()
+  const { header, footer } = childVhd
 
   const parentName = parentPath.split('/').pop()
   const parentUuid = parentVhd.footer.uuid
   if (
     header.parentUnicodeName !== parentName ||
     !isEqual(header.parentUuid, parentUuid)
   ) {
+    footer.diskType = 4
```

Comment:
Why?

Reply:
it allows for creating chains from thin air during testing

Reply:
Maybe it should be opt-in:

```js
if (footer.diskType !== HARD_DISK_TYPE_DIFFERENCING) {
  if (!force) {
    throw new Error('cannot chain disk of type '+ footer.diskType)
  }
  footer.diskType = HARD_DISK_TYPE_DIFFERENCING
}
```

What do you think?